config/settings.yml

```diff
@@ -0,0 +1,59 @@
+# This is the central repository for settings in Champaign. Settings are defined in the following files:
+#
+#    config/settings.yml
+#    config/settings/#{environment}.yml
+#    config/environments/#{environment}.yml
+#
+#    config/settings.local.yml
+#    config/settings/#{environment}.local.yml
+#    config/environments/#{environment}.local.yml
+#
+# Settings defined in files lower on the list override settings higher on the list. So, settings in
+# config/settings/#{environment}.yml will override the settings in this file, and settings in
+# config/settings/#{environment}.local.yml will override the settings in that file.
+
+# General Champaign settings.
+     secret_key_base: 'Set this in settings.local.yml or in an environment settings file.'
```

Comment:
Uncommented lines need to be present for the application to start up at all or contain values (like in the liquid source) which can be safely included without fear for someone discovering their content.